src/Defer/Defer.php

```diff
@@ -74,17 +74,17 @@ public function __construct(CanStreamResponse $stream, GraphQL $graphQL)
     /**
      * Set the tracing directive on all fields of the query to enable tracing them.
      *
-     * @param  \Nuwave\Lighthouse\Events\ManipulateAST  $ManipulateAST
+     * @param  \Nuwave\Lighthouse\Events\ManipulateAST  $manipulateAST
      * @return void
      */
-    public function handleManipulateAST(ManipulateAST $ManipulateAST): void
+    public function handleManipulateAST(ManipulateAST $manipulateAST): void
```

Comment:
Maybe we should add in our contribution guide how variable naming is. Should this be named `manipulateAst` instead. For example in the `GraphQl.php` file we have a variable called `astBuilder`, if the naming should be the same as in here, it should be named `aSTBuilder`.

Reply:
`aSTBuilder` just looks really weird to me. Variable naming is not that important, we just camelCase and call it a day :)

Reply:
I agree, aSTBuilder looks really weird and is confusing. It looks like we are saying `a ST Builder`, not `AST Builder`. 
But great, sound perfect :)